### ISSUE TITLE
Prob galaxy is use in decoder only to check whether it is bigger than 0 

### ISSUE THREAD

**Issue by ismael-mendoza** [prob-ml/bliss#384]
This looks a bit strange to me, but not sure if there is a better alternative. Perhaps checking whether the `autoencoder_ckpt` is not `None`? 

https://github.com/prob-ml/bliss/blob/3f30a40f2cbb64dc5a91497017aa013fc21651c3/bliss/models/decoder.py#L83

**Comment by dereklhansen** [comment]
Good catch. "prob_galaxy" is a hyperparameter that governs the prior, so I agree the decoder shouldn't need to know it.

> Perhaps checking whether the autoencoder_ckpt is not None?

This sounds good to me.
